app/scripts/modules/core/src/presentation/forms/FormikFormField.tsx

```diff
@@ -56,27 +67,41 @@ export class FormikFormField<T = any> extends React.Component<IFormikFormFieldPr
     }));
   };
 
+  public name = () => this.props.name;
+
+  public label = () => ifString(this.props.label);
```

Comment:
Isn't `this.props.label` required to be a string?

Reply:
It can be any `React.Node` including JSX